go/vt/vtgate/engine/route_test.go

```diff
@@ -82,7 +82,7 @@ func TestSelectInformationSchemaWithTableAndSchemaWithRoutedTables(t *testing.T)
 	stringListToExprList := func(in []string) []evalengine.Expr {
 		var schema []evalengine.Expr
 		for _, s := range in {
-			schema = append(schema, evalengine.NewLiteralString([]byte(s)))
+			schema = append(schema, evalengine.NewLiteralString([]byte(s), 0))
```

Comment:
all the magic 0 in this files should be `collations.Unknown`, right?

Reply:
yes it should